glue/plugins/ginga_viewer/__init__.py

```diff
@@ -0,0 +1,10 @@
+try:
+    from .client import *
+    from .qt_widget import *
+except ImportError:
+    import warnings
+    warnings.warn("Could not import ginga plugin, since ginga is required")
```

Comment:
I created a centralized logger (in ``glue.logger``) and also added a ``--verbose`` option to the glue command-line script so that we can see INFO messages (default is WARN+)